.github/workflows/create-evaluation_reports.yml

```
@@ -139,3 +147,4 @@ jobs:
           ${{ needs.read_inputs.outputs.models_table }}
 #        reviewers: ${{ github.actor }}
 
+
```

YAMLlint (1.38.0)

[warning] too many blank lines (2 > 0)

(empty-lines)